app/assets/stylesheets/resets.scss

```diff
@@ -139,7 +139,7 @@ p {
 	padding: 5px 0;
 	border-radius: 6px;
 	position: absolute;
-  right: 43px;
+  right: -5px;
```

Comment:
Okay this is cool but just in general, why are we putting this stuff in CSS reset styles? Doesn't have to be addressed in this PR but that's gross imo :sweat_smile:

Reply:
Agreed. We're doing some rework of the branding, and with that'll come some rework of the scss... things like this should be in a global.scss or something like that.